frontend/BackPressHandler.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
```

Comment:
Where is this file used?

Reply:
it isnt, thought i would need it but dont for the time being so I will delete it